client/js/components/assessments/item.jsx

```diff
@@ -9,12 +9,14 @@ export default class Item extends React.Component{
 
   static propTypes = {
     question                   : React.PropTypes.object.isRequired,
+    response                   : React.PropTypes.array.isRequired,
```

Comment:
Let's start documenting these.  Let's try an inline comment at the end of the line for now.  Also, close up the spacing some, but keep things aligned.